### PR TITLE
Chore/update particle network

### PR DESCRIPTION
**Pull request by soos3d**
### Description

A simple update to the Particle tutorial

### CODE REVIEW

**Comment by albertov19** [comment]
Hey @soos3d  - Currently @themacexpert is in Devcon. Apologies for the delay in the review

**Comment by soos3d** [comment]
> Hey @soos3d - Currently @themacexpert is in Devcon. Apologies for the delay in the review

No worries, just making sure we don't forget about it :)

File: builders/integrations/wallets/particle-network.md [hunk 1]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
 
 ```js
 --8<-- 'code/builders/integrations/wallets/particle/example-app.js'
 ```
 
-The full demo repository containing the above code can be found in the [particle-moonbeam-demo GitHub repository](https://github.com/TABASCOatw/particle-moonbeam-demo){target=\_blank}.
-
 That concludes the brief introduction to Particle's Smart Wallet-as-a-Service stack and how to get started with Particle on Moonbeam. For more information, you can check out [Particle Network's documentation](https://docs.particle.network){target=\_blank}.
 
+Find the repository with the full code implementation on the [Particle Network GitHub](https://github.com/Particle-Network/connect-moonbeam-tutorial).
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:136]:
```suggestion
Find the repository with the full code implementation on the [Particle Network GitHub](https://github.com/Particle-Network/connect-moonbeam-tutorial){target=\_blank}.
```

File: builders/integrations/wallets/particle-network.md [hunk 2]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
-If you'd like to natively use ERC-4337 AA, also install `@particle-network/aa`.
+To start, we’ll configure and initialize Particle Connect (Particle's flagship authentication SDK). Begin by creating a new file called `ConnectKit.tsx` in your project’s root directory, where we’ll set up the `ParticleConnectKit` component as the main interface for configuration.
 
-=== "npm"
+Before proceeding, head back to the [Particle dashboard](https://dashboard.particle.network){target=\_blank} and retrieve the following API keys:
 
-    ```bash
-    npm install @particle-network/aa
-    ```
+- **`projectId`** – your project’s unique ID.
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:67]:
```suggestion
- **`projectId`** – your project’s unique ID
```

File: builders/integrations/wallets/particle-network.md [hunk 3]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
+- **`clientKey`** – your client-specific key.
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:68]:
```suggestion
- **`clientKey`** – your client-specific key
```

File: builders/integrations/wallets/particle-network.md [hunk 4]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
+- **`appId`** – your application ID.
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:69]:
```suggestion
- **`appId`** – your application ID
```

File: builders/integrations/wallets/particle-network.md [hunk 5]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
 
 ## Install Dependencies {: #install-dependencies }
 
-To integrate Particle Network within your Moonbeam application, you'll need to install a number of dependencies, the specifics of which depend on whether you intend on purely using the default EOA generated by Particle's Wallet-as-a-Service, or if you intend to leverage an attached smart account.
+You'll need only a few dependencies to integrate Particle Connect into your Moonbeam application. Particle Connect offers built-in Account Abstraction (AA) support; however, in this example, we'll install the Particle AA SDK to utilize EIP-1193 providers, such as ethers.
 
-For both EOA and smart account utilization, install `@particle-network/auth`.
-
-=== "npm"
+=== "yarn"
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:53]:
```suggestion
```

File: builders/integrations/wallets/particle-network.md [hunk 6]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
 
     ```bash
-    npm install @particle-network/auth
+    yarn add @particle-network/connectkit viem@^2 @particle-network/aa ethers
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:56]:
```suggestion
 yarn add @particle-network/connectkit viem@^2 @particle-network/aa ethers
```

File: builders/integrations/wallets/particle-network.md [hunk 7]
```diff
@@ -3,30 +3,28 @@ title: Add Particle Network to a dApp
...
+Key components of Particle Network's Moonbeam integration include:
 
-- Particle Network [Wallet-as-a-Service](https://docs.particle.network/getting-started/smart-wallet-as-a-service){target=\_blank} - this is the flagship Wallet-as-a-Service product offered by Particle Network, enabling application-embedded wallets powered by MPC-TSS to facilitate a seamless, Web2-like onboarding and interaction experience
-- Particle Network Modular [AA Stack](https://docs.particle.network/developers/account-abstraction){target=\_blank} - beyond the base EOA-centric interaction that happens by default through Particle's Wallet-as-a-Service, Particle also has a native modular AA stack for the implementation of ERC-4337 account abstraction on Moonbeam. This means inherent flexibility with the smart account, bundler, and paymaster you use in tandem with Particle's Wallet-as-a-Service when building AA-enabled applications
+- **Particle Connect**: Particle's flagship Wallet-as-a-Service solution, offering embedded wallets powered by MPC-TSS for smooth, Web2-like onboarding and interactions, with Account Abstraction support integrated within a single SDK.
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:18]:
```suggestion
- **Particle Connect**: Particle's flagship Wallet-as-a-Service solution, offering embedded wallets powered by MPC-TSS for smooth, Web2-like onboarding and interactions, with Account Abstraction support integrated within a single SDK
```

File: builders/integrations/wallets/particle-network.md [hunk 8]
```diff
@@ -3,30 +3,28 @@ title: Add Particle Network to a dApp
...
+- **Particle Network Modular [AA Stack](https://docs.particle.network/developers/account-abstraction){target=\_blank}**: Beyond the default EOA-based interactions, Particle also offers a modular AA stack for ERC-4337 account abstraction on Moonbeam, allowing flexibility in the smart account, bundler, and paymaster configurations to suit AA-enabled applications.
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:19]:
```suggestion
- **Particle Network Modular [AA Stack](https://docs.particle.network/developers/account-abstraction){target=\_blank}**: Beyond the default EOA-based interactions, Particle also offers a modular AA stack for ERC-4337 account abstraction on Moonbeam, allowing flexibility in the smart account, bundler, and paymaster configurations to suit AA-enabled applications
```

File: builders/integrations/wallets/particle-network.md [hunk 9]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
+These keys are essential as they connect your Particle Connect instance with the Particle dashboard, enabling features like no-code customization, user activity tracking, and API request authentication.
 
-    ```bash
-    yarn add @particle-network/aa
-    ```
+Place the API keys in a `.env` file in the following format:
 
-When creating an AA, you'll need to pass it a provider. You can use a Particle Provider or any EVM wallet provider. To use a Particle Provider, you'll need to install `@particle-network/provider`.
+```shell
+NEXT_PUBLIC_PROJECT_ID='PROJECT_ID'
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:76]:
```suggestion
NEXT_PUBLIC_PROJECT_ID='INSERT_PROJECT_ID'
```

File: builders/integrations/wallets/particle-network.md [hunk 10]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
+NEXT_PUBLIC_CLIENT_KEY='CLIENT_KEY'
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:77]:
```suggestion
NEXT_PUBLIC_CLIENT_KEY='INSERT_CLIENT_KEY'
```

File: builders/integrations/wallets/particle-network.md [hunk 11]
```diff
@@ -50,117 +48,91 @@ To use Particle Network's Smart Wallet-as-a-Service on Moonbeam, you'll need to
...
+NEXT_PUBLIC_APP_ID='APP_ID'
```
Comment (themacexpert) [inline builders/integrations/wallets/particle-network.md:78]:
```suggestion
NEXT_PUBLIC_APP_ID='INSERT_APP_ID'
```

**Review by themacexpert — CHANGES_REQUESTED** [review]
Thanks for PR! This looks great. Just some comments here to align this with our format guidelines. Once these are in, I'll give it a final check and we'll be good to go.

**Comment by themacexpert** [comment]
> gmgm, any chance we can review this? Thank you!

Thanks for the PR!

**Comment by soos3d** [comment]
> Thanks for PR! This looks great. Just some comments here to align this with our format guidelines. Once these are in, I'll give it a final check and we'll be good to go.

sounds good! I'll take care of it now

**Comment by soos3d** [comment]
Reviewed and accepted the suggestions!

**Comment by themacexpert** [comment]
Screenshot 2024-11-18 at 9 13 15 PM
If you can sort out this bash command then we should be good to go! thx  @soos3d

**Comment by soos3d** [comment]
> Screenshot 2024-11-18 at 9 13 15 PM If you can sort out this bash command then we should be good to go! thx @soos3d

Sorry about that, this last commit should fix it. Please let me know!

**Review by themacexpert — APPROVED** [review]

**Pull request merged**